File: 기초 사고력문제/백준_4344_평균은 넘겠지/평균은 넘겠지.cs
using System;
using static System.Console;

class Program
{
    public static void Main(string[] args)
    {
        int n = int.Parse(ReadLine());
        while (n-- > 0)
        {
            string[] s = ReadLine().Split();
            int m = int.Parse(s[0]);
            int[] arr = new int[m];
            int ever = 0;
            for (int i = 0; i < m; i++)
            {
                arr[i] = int.Parse(s[i+1]);
                ever += arr[i];
            }
            ever /= m;
            int cnt = 0;
            for (int i = 0; i < arr.Length; i++)
                if (arr[i] > ever) cnt++;
            
            WriteLine("{0:F3}%", ((float)cnt*100/m));
        }
        
    }
}

File: 기초 사고력문제/백준_4344_평균은 넘겠지/평균은 넘겠지.py
n = int(input())

while n > 0:
    n-=1
    arr = list(map(int, input().split()))
    ever = 0
    for i in range(1,arr[0]+1):
        ever+= arr[i]

    ever /= arr[0]
    cnt = 0
    for i in range(1,arr[0]+1):
        if arr[i] > ever : cnt+=1

    print("%.3f%%" % (cnt*100/arr[0]))

